### ISSUE TITLE
PAT duplicate measure names don't increment

### ISSUE THREAD

**Issue by kflemin** [NREL/OpenStudio-PAT#18]
_From @shorowit on April 13, 2017 15:36_

When you duplicate a measure Foo in PAT 2.1.0, you get "Foo 2". If you then duplicate "Foo 2", you get "Foo 2 2" instead of "Foo 3". It does work correctly if you duplicate measure "Foo" twice.

@asparke2 

_Copied from original issue: NREL/OpenStudio#2570_

**Comment by kflemin** [comment]
_From @shorowit on April 13, 2017 15:44_

I should clarify that this is in algorithmic mode.

**Comment by kflemin** [comment]
_From @DavidGoldwasser on April 14, 2017 5:18_

Behavior is same in manual model

**Comment by kflemin** [comment]
_From @rHorsey on July 10, 2017 20:52_

Confirmed fixed in 2.2.0

**Comment by kflemin** [comment]
_From @shorowit on August 18, 2017 12:0_

This is not fixed. Screenshot from PAT 2.2.1:
![image](https://user-images.githubusercontent.com/5861765/29458059-7f20466e-83da-11e7-81c4-13ee946c9bf0.png)

**Comment by kflemin** [comment]
This is by design.  If you want to increment the numbers, you'd have to duplicate the original measure only, and not copies of the original measure.

**Comment by shorowit** [comment]
In my opinion, it seems unreasonable to expect a user to do this. If you are trying to make many copies at once, it's worth noting that when you duplicate the measure, PAT immediately takes you to the duplicated measure. So this means you have to scroll back up to the top each time to duplicate the original measure again.

That said, I wouldn't care too much about this if https://github.com/NREL/OpenStudio-PAT/issues/19 was implemented. That is really my desire.

**Comment by kflemin** [comment]
Yes, I think #19 is a better alternative.  I'll work on that one first.
